docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3455,6 +3455,14 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -ruler.max-rule-groups-per-tenant-by-namespace
 [ruler_max_rule_groups_per_tenant_by_namespace: <map of string to int> | default = {}]
 
+# (experimental) List of namespaces that are protected from modification unless
+# a special HTTP header is used. If a namespace is protected, it cannot be
+# modified via the ruler's config API only read. The value is a list of strings,
```

Comment:
```suggestion
# a special HTTP header is used. If a namespace is protected, it can only be read, not modified,
# via the ruler's configuration API. The value is a list of strings,
```

Reply:
Done in https://github.com/grafana/mimir/pull/8444/commits/b83ba6cd8519aa206c6af52ba4cd522813e3f6a5 - FYI for the future, these docs are auto-generated. The text to change should always be on the file named `valiation/limits.go`